llama-index-integrations/llms/llama-index-llms-gaudi/pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.gaudi"
+
+[tool.llamahub.class_authors]
+GaudiLLM = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.10"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index llms gaudi integration"
+license = "MIT"
+name = "llama-index-llms-gaudi"
+packages = [{include = "llama_index/"}]
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+huggingface-hub = "^0.23.0"
+torch = "^2.1.2"
+text-generation = "^0.7.0"
+llama-index-core = "^0.11.0"
```

Comment:
`llama-index-llms-huggingface` is imported but not listed as a dependency

Reply:
Updated the pyproject.toml file to include llama-index-llms-huggingface